src/plugins/intel_cpu/src/nodes/executors/fullyconnected_config.hpp

```diff
@@ -28,34 +25,6 @@ struct FCAttrs {
     // @todo should be memory desc or vector dims
     MemoryCPtr decompressionSubtractPtr;
     MemoryCPtr decompressionMultiplyPtr;
```

Comment:
What prevents you from implementing this proposal? At least the formulation sounds reasonable.

Reply:
Handling of the optional memory will be a leftover.